api/helpers/errors.js

```diff
@@ -58,6 +58,26 @@ class ArgumentIsNotIntError extends Error {
     }
 }
 
+class ArgumentIsNotNumberError extends Error {
```

Comment:
yah, but this is just for sake of being more specific when receiving an error - price is not supposed to be INT only, it can be double or whatever, so ArgumentIsNotIntError is not very correct in this case

Reply:
merge this two functions into something like ArgumentIsNotCorrectType.
hehz.